dashboard/app/views/levels/editors/_collapsed.html.haml

```diff
@@ -0,0 +1,9 @@
+- xml_id = "#{xml_id}"
+- collapse_xml = "#{xml_id}_collapse_xml"
+
+.min_max_label{data: {toggle: "collapse", target: "##{collapse_xml}"}}
+  Open/Close
+.row.collapse.in{id: collapse_xml}
+  .span8
+    ~ f.text_area :start_blocks, placeholder: "#{xml_id}" + ' Blocks', rows: 4, value: @level.pretty_block("#{xml_id}")
```

Comment:
nit: since this partial is doing more than _just_ collapsing, can you give it a more descriptive name? Something like `collapsible_block_editor`